schema/sql/api.sql

```diff
@@ -127,6 +142,7 @@ comment on column v1.ssh_public_key.uid is
     $$User ID the key is currently linked to$$;
 alter view v1."ssh_public_key" owner to api;
 grant select on table v1."ssh_public_key" to "api-anon";
+grant update,insert on table v1."ssh_public_key" to "api-user-manage";
```

Comment:
Wouldn't you need `delete` as well?

Reply:
good catch. Don't do deleting yet, but I will.